CMakeLists.txt

```diff
@@ -11,16 +11,16 @@ find_package(GTest REQUIRED)
 
 include(CTest)
 
-set(SRC_LIST "src/DBConnection.cpp" "src/DBConnection.hpp")
+set(SRC_LIST "src/DBConnection.cpp" "src/DBConnection.hpp" src/DBConnection2.cpp src/DBConnection2.hpp src/ConnectionDispatcher.cpp src/ConnectionDispatcher.hpp)
 
-add_executable(testrunner "${SRC_LIST}" test/ConnectionTest.cpp)
+add_executable(testrunner "${SRC_LIST}" test/ConnectionTest.cpp test/ConnectionTest2.cpp)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 target_include_directories(testrunner PRIVATE src test)
 
 target_compile_options(testrunner PRIVATE -Wextra -pedantic)
 
-target_link_libraries(testrunner PRIVATE gtest gtest_main)
+target_link_libraries(testrunner PRIVATE gtest gtest_main ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
@morheit, what is the purpose of var usage?

Reply:
@malirod , we need this var because [Google Test uses threads](https://github.com/google/googletest/blob/master/googletest/README.md).

Reply:
@malirod , var removed